pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTThrowStatement.java

```diff
@@ -32,23 +32,4 @@ public ASTExpression getExpr() {
         return (ASTExpression) getFirstChild();
     }
 
-    /**
-     * Gets the image of the first ASTClassOrInterfaceType child or
-     * <code>null</code> if none is found. Note that when the statement is
-     * something like throw new Exception, this method returns 'Exception' and
-     * if the throw statement is like throw e: this method returns 'e'. A
-     * special case of returning <code>null</code> is when the throws is like
-     * throw this.e or throw this.
-     *
-     * This is too specific
-     *
-     * <p>TODO - use symbol table (?)</p>
-     *
-     * @return the image of the first ASTClassOrInterfaceType node found or
-     *     <code>null</code>
-     */
-    public String getFirstClassOrInterfaceTypeImage() {
```

Comment:
I guess, we should deprecate this method on master. It seems to be used only by one rule - ExceptionAsFlowControl

Reply:
Done: https://github.com/pmd/pmd/commit/a5eb60243dfa12d3ca2cc8eb3bfff091654c66ab